cms/static/js/models/course_update.js

```diff
@@ -6,6 +6,14 @@ define(["backbone", "jquery", "jquery.ui"], function(Backbone, $) {
             "content" : "",
             "push_notification_enabled": false,
             "push_notification_selected" : false
+        },
+        validate: function(attrs) {
+            var date_exists = (attrs.date !== null && attrs.date !== "");
+            var date_is_valid_string = ($.datepicker.formatDate('MM d, yy', new Date(attrs.date)) === attrs.date);
```

Comment:
I'm confused about date_is_valid_string and date_is_date. Is date_is_date redundant?